scootapi/scoot.thrift

```diff
@@ -31,8 +35,9 @@ struct JobDefinition {
   2: optional JobType jobType,
 }
 
+// TODO(dbentley): rename this to... Exec? CloudExec? CloudScoot?
 service Proc {
-  Job RunJob(1: JobDefinition job) throws (
+   JobId RunJob(1: JobDefinition job) throws (
```

Comment:
All of the job info is about to disappear from the API (as it sits in the queue). I thought it would be more consistent to return just the ID.

TaskIds: I don't understand why task IDs are optional in job definition. I see the value of them being:
mandatory (the client has to fill them out)
not allowed in a definition (the server fills them out)
absent entirely (just use the index into the list of tasks).